diff --git a/vispy/util/fonts/_freetype.py b/vispy/util/fonts/_freetype.py
index 77ad458..d0e104b 100755
--- a/vispy/util/fonts/_freetype.py
+++ b/vispy/util/fonts/_freetype.py
@@ -12,12 +12,12 @@ import numpy as np
 
 # Convert face to filename
 from ._vispy_fonts import _vispy_fonts, _get_vispy_font_filename
-if sys.platform.startswith('linux'):
-    from ...ext.fontconfig import find_font
-elif sys.platform.startswith('win'):
-    from ._win32 import find_font  # noqa, analysis:ignore
-else:
-    raise NotImplementedError
+# if sys.platform.startswith('linux'):
+#     from ...ext.fontconfig import find_font
+# elif sys.platform.startswith('win'):
+#     from ._win32 import find_font  # noqa, analysis:ignore
+# else:
+#     raise NotImplementedError
 
 _font_dict = {}
 
@@ -44,6 +44,7 @@ def _load_font(face, bold, italic):
 
 def _load_glyph(f, char, glyphs_dict):
     """Load glyph from font into dict"""
+    return
     from freetype import (FT_LOAD_RENDER, FT_LOAD_NO_HINTING,
                           FT_LOAD_NO_AUTOHINT)
     flags = FT_LOAD_RENDER | FT_LOAD_NO_HINTING | FT_LOAD_NO_AUTOHINT
